woocommerce/woocommerce
# Fix SVG markup in miniCartIcon

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Picked from https://github.com/woocommerce/woocommerce/pull/44011 (added changelog, testing instructions, and allowing CI to run).

Closes #44011
Closes #44006



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Add Mini Cart Block to a page
2. Change the icon to "bag" and save changes
3. View frontend. Confirm icon renders correctly



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
